### PR TITLE
Style pass

### PR DESCRIPTION
**Pull request by SJShaw**
Fixes the majority of style issues found via pylint and flake8 that aren't variants of `too-many-*`. 

Some style messages were common enough to justify a single commit for that specific type.

### CODE REVIEW

**Review by kblin — CHANGES_REQUESTED** [review]
I think cleaning up the lasso and thiopeptide lists like the lanthipeptide and sactipeptide modules would be nice, otherwise I think it's good to go.

File: antismash/modules/lassopeptides/specific_analysis.py [hunk 1]
```diff
@@ -518,70 +518,70 @@ def generate_rodeo_svm_csv(record: Record, query: CDSFeature, leader: str, core:
     columns.append(core[:9].count("P"))
     # Estimated core charge
     charge_dict = {"E": -1, "D": -1, "K": 1, "H": 1, "R": 1}
-    columns.append(sum([charge_dict[aa] for aa in core if aa in charge_dict]))
```
[the remark this answers is not in the document]
Comment (SJShaw) [reply antismash/modules/lassopeptides/specific_analysis.py:520]: Done

File: antismash/modules/thiopeptides/rodeo.py [hunk 2]
```diff
@@ -177,49 +177,49 @@ def generate_rodeo_svm_csv(leader: str, core: str, previously_gathered_tabs: Lis
     # Ratio of length of leader / length of core
     columns.append(len(core) / len(leader))
     # Ratio of heterocyclizable residues / length of core
-    columns.append(sum([core.count(aa) for aa in "CST"]) / len(core))
+    columns.append(sum(core.count(aa) for aa in "CST") / len(core))
     # Number in leader of each amino acid
     columns += [leader.count(aa) for aa in "ARDNCQEGHILKMFPSTWYV"]
     # Aromatics in leader
-    columns.append(sum([leader.count(aa) for aa in "FWY"]))
```
[the remark this answers is not in the document]
Comment (SJShaw) [reply antismash/modules/thiopeptides/rodeo.py:184]: Done

File: antismash/common/layers.py [hunk 3]
```diff
@@ -66,21 +66,21 @@ def __getattr__(self, attr: str) -> Any:
...
+        current_id = f"<strong>{self.seq_record.id}</strong>"
 
         orig_id = self.seq_record.original_id
 
         if orig_id:
             if len(orig_id) < 40:
-                current_id += " (original name was: %s)" % orig_id
+                current_id += f" (original name was: {orig_id})"
             else:
-                current_id += " (original name was: %s...)" % orig_id[:60]
```
[the remark this answers is not in the document]
Comment (kblin) [reply antismash/common/layers.py:77]: Scheduled for a future PR

**Review by kblin — APPROVED** [review]
Looks good to me 👍

**Pull request merged**